src/invert.test.ts

```diff
@@ -0,0 +1,55 @@
+import { invert } from './invert';
+import { pipe } from './pipe';
+
+describe('invert', function () {
```

Comment:
No need to wrap the whole test suite with a describe, it'll show up as the file name. Use describe when you have more than one category of tests.

Reply:
✅ Done